packages/preview2-shim/lib/synckit/index.js

```diff
@@ -87,7 +87,7 @@ export function createSyncFn(workerPath, debug, callbackHandler) {
     }
     return result;
   };
-  worker.unref();
+  // worker.unref();
```

Comment:
```suggestion
  if (worker.unref) worker.unref();
```

Reply:
yes -- that's a much nicer solution! :)

Nevertheless, I think, it's not enough!

The timeouts in the CI pipeline are most likely caused by hanging processes affected by this particular change resp. broken termination of workers. :(

We have to check for active event listeners or timers...